hazelcast/include/hazelcast/client/sql/impl/query_id.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cstdint>
+
+#include "hazelcast/util/export.h"
+
+namespace hazelcast {
+namespace client {
+namespace sql {
+namespace impl {
+
+class HAZELCAST_API query_id
+{
+public:
+    query_id(int64_t member_id_high,
+             int64_t member_id_low,
+             int64_t local_id_high,
+             int64_t local_id_low);
+
+    int64_t member_id_high() const;
+    int64_t member_id_low() const;
```

Comment:
This is how Java uses uuid, but we can use boost::uuid which is 16 bytes instead.